CWP/papers/roadmap/authors/a2tex.py

```diff
@@ -0,0 +1,185 @@
+#! /usr/bin/env python
```

Comment:
Can you change this to "#! /usr/bin/env python3"? On systems with 2 and 3 "python" usually resolves to python2 for backward compatibility.

Reply:
Done

Reply:
But after checking with a 2.x version, my check doesn't work because the print syntax used to write in a file other than stdout is not valid in 2.x (and vice-versa, the 2.x one doesn't work in 3.x...)... I guess that for this script this is not so critical, I'm reluctant to put a Python interpreter that will not work on some machines despite it is a valid one (like on my machine!).

Reply:
OK, fine then - not a lot of people need to run this.